__tests__/server.test.js

```diff
@@ -128,3 +128,57 @@ describe("/api/articles/:article_id", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET request", () => {
+    test("200: should return the an array of comments for the requested article", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeArrayOfSize(11);
+          comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: 1,
```

Comment:
Really nicely done - this is an extra bit of security that a lot of people often forget about, but we absolutely expect the comments to all have article_id 1 in this case 👏